Add CI process in the README (#73)
epinio/epinio-end-to-end-tests: README.md

[![Scenario 1](https://github.com/epinio/epinio-end-to-end-tests/actions/workflows/scenario_1_cypress_chrome.yml/badge.svg)](https://github.com/epinio/epinio-end-to-end-tests/actions/workflows/scenario_1_cypress_chrome.yml)
[![Scenario 2](https://github.com/epinio/epinio-end-to-end-tests/actions/workflows/scenario_2_cypress_chrome.yml/badge.svg)](https://github.com/epinio/epinio-end-to-end-tests/actions/workflows/scenario_2_cypress_chrome.yml)
# epinio-end-to-end-tests
This repository contains all the files necessary to run Epinio end-to-end tests.

In the cypress directory are stored the tests written using the [Cypress](https://www.cypress.io/) testing framework.

Tests are executed every night in the CI. For now, two scenarios are tested.
### Scenario 1
In this first scenario, Epinio is deployed with default options. </br>
You can check all the things we test directly in the [file](./cypress/integration/scenarios/install_with_default_options.spec.ts).

### Scenario 2

> Second scenario tests Epinio installation with s3 and external registry configured. </br>

Reviewer: ```suggestion
Second scenario tests Epinio installation with S3 and external registry configured. </br>
```

> Contrary to the first scenario, we only play a small bunch of [tests](./cypress/integration/scenarios/install_with_s3_and_external_registry.spec.ts).

Reviewer: ```suggestion
Unlike the first scenario, we only play a small bunch of [tests](./cypress/integration/scenarios/install_with_s3_and_external_registry.spec.ts).
```

> At the moment, both scenarios are executed on Chrome. </br>

Reviewer: ```suggestion
For now, both scenarios are executed on Chrome. </br>
```

Later, we will execute Scenario 2 on Firefox instead.
## Running the tests

> __Attention__, Epinio is using Rancher UI but Epinio will get its own UI in the coming days.

Reviewer: Maybe better to say `weeks` if a user read this ;-)
```suggestion
__Attention__, Epinio is using Rancher UI but Epinio will get its own UI in the coming weeks.
```
